features/kafka/producer/src/main/java/org/opennms/features/kafka/producer/OpennmsKafkaProducer.java

```diff
@@ -408,6 +409,15 @@ private void processKafkaSendQueue() {
                     producer.send(producerRecord, (recordMetadata, e) -> {
```

Comment:
Do we need to worry about the exceptions that `send()` itself might throw (caught below)? Or are we only concerned about the exceptions that will be given to us in the callback.

Reply:
Most of them are not recoverable exceptions and all of them are extended from RuntimeException so I would assume that's okay.